### PR TITLE
[IMP] *: remove useless comments from po/pot files

### PR DESCRIPTION
**Pull request by oomsveta**
Clean up of WEB_TRANSLATION_COMMENT and python-format from the po(t) files.

The `python-format` flag specifies the "flavor" of format specifier to use, but our code only uses this format and doesn't check the flag at all, making it useless.

The variable WEB_TRANSLATION_COMMENT contains the string "openerp-web" which was used in po/pot files to flag translations to be used on the client side, but has been replaced by JAVASCRIPT_TRANSLATION_COMMENT since odoo#105589.

This pull request:
- Removes WEB_TRANSLATION_COMMENT from the code
- Re-exports .pot files that still contain openerp-web
- `msgmerge`s po files that still contain openerp-web
- Removes the obsolete entries created by msgmerge
- Removes every occurrence of python-format

Task-3599456.

Enterprise: https://github.com/odoo/enterprise/pull/51147
Design Themes: https://github.com/odoo/design-themes/pull/749

### CODE REVIEW

File: addons/website_sale_stock/i18n/pt_BR.po [hunk 1]
```diff
@@ -181,24 +187,37 @@ msgid "Show availability Qty"
 msgstr "Mostrar Quant disponível"
 
 #. module: website_sale_stock
+#. odoo-python
 #: code:addons/website_sale_stock/models/sale_order.py:0
 #, python-format
```
Comment (HydrionBurst) [inline addons/website_sale_stock/i18n/pt_BR.po:192]: You can also remove this one
https://github.com/odoo/odoo/blob/eab9453a0dff21fb3672a05eed346e073563a5a6/odoo/tools/translate.py#L811-L813

**Comment by mart-e** [comment]
@robodoo override=ci/security override=ci/style
Reason: too many files

**Review by HydrionBurst — APPROVED** [review]
LGTM

**Comment by mart-e** [comment]
@robodoo delegate+

**Comment by oomsveta** [comment]
robodoo r+

**Comment by oomsveta** [comment]
robodoo rebase-ff

**Pull request closed without merging**